docs/api/hardhat/getting-started.md

```diff
@@ -2,28 +2,27 @@
 
 [Hardhat](https://hardhat.org) is an Ethereum development environment, designed for easy smart contract development in Solidity. One of its most prominent features is extendability: you can easily add new plugins to your hardhat project.
 
-zkSync has three plugins for hardhat:
+zkSync has three plugins for Hardhat:
 
-- `@matterlabs/hardhat-zksync-solc` for smart contract compilation using Solidity.
-- `@matterlabs/hardhat-zksync-vyper` for smart contract compilation using Vyper.
-- `@matterlabs/hardhat-zksync-deploy` for smart contract deployment.
+- [@matterlabs/hardhat-zksync-solc](./plugins.md#matterlabs-hardhat-zksync-solc) for smart contract compilation using Solidity.
+- [@matterlabs/hardhat-zksync-vyper](./plugins.md#matterlabs-hardhat-zksync-vyper) for smart contract compilation using Vyper.
+- [@matterlabs/hardhat-zksync-deploy](./plugins.md#matterlabs-hardhat-zksync-deploy) for smart contract deployment.
```

Comment:
Can you add a hyphen(-) and add some prefix actionable words(e.g - Is used ....)?
E.g 
- [@matterlabs/hardhat-zksync-solc](./plugins.md#matterlabs-hardhat-zksync-solc) - Is used for smart contract compilation using Solidity.
- [@matterlabs/hardhat-zksync-vyper](./plugins.md#matterlabs-hardhat-zksync-vyper) - Is used for smart contract compilation using Vyper.
- [@matterlabs/hardhat-zksync-deploy](./plugins.md#matterlabs-hardhat-zksync-deploy) - Is used for smart contract deployment.